src/UnlockAccountController.js

```diff
@@ -99,14 +99,15 @@ function (Okta, FormController, Enums, FormType, ValidationUtil, ContactSupport,
         }
         else {
           formChildren.push(FormType.Input({
-            placeholder: Okta.loc('account.unlock.email.or.username.placeholder', 'login'),
+            label: Okta.loc('account.unlock.email.or.username.placeholder', 'login'),
+            'label-top': true,
             name: 'username',
             input: TextBox,
+            inputId: 'account-recovery-username',
```

Comment:
curious why need to hard code inputId?

Reply:
To have a handy selector for testing. Without hard coding that, id will be generated, example: "input66"